.github/workflows/tail-call.yml

```diff
@@ -75,23 +77,25 @@ jobs:
 
       - name: Native Windows (debug)
         if: runner.os == 'Windows' && matrix.architecture != 'ARM64'
+        shell: cmd
         run: |
-          choco install llvm --allow-downgrade --no-progress --version ${{ matrix.llvm }}.1.0
-          ./PCbuild/build.bat --tail-call-interp -d -p ${{ matrix.architecture }}
+          choco install llvm --allow-downgrade --no-progress --version ${{ matrix.llvm }}.1.5
+          ./PCbuild/build.bat --tail-call-interp -d -p ${{ matrix.architecture }} "/p:PlatformToolset=clangcl" "/p:LLVMToolsVersion=19.1.5" "/p:LLVMInstallDir=C:\Program Files\LLVM"
```

Comment:
It might be convenient to add `--clang` as a build.bat option, though you'd still need the quoting to get the other variables in.

Also, FWIW, unless you're explicitly trying to override other settings, you can set these variables as environment variables instead. The MSBuild files can override those if it isn't authored properly, but at least `PlatformToolset` is correct, and I expect the other two will be too.

Reply:
Actually, I was thinking of using the default clang that comes with VS. The latest VS comes with Clang 19, so it's perfect for this. However, I don't know if the CI runners updated to use the latest VS yet, and I don't want a situation where sometimes we break on older VS and sometimes we don't. So for now, I'll let users manually specify, and maybe in a year or so we can revisit adding a `--clang` option.